src/item.h

```diff
@@ -1294,11 +1294,11 @@ class item : public visitable
          * @param dmg_type The type of incoming damage
          * @param to_self If this is true, it returns item's own resistance, not one it gives to wearer.
          * @param bodypart_target The bodypart_id or sub_bodypart_id of the target bodypart.
-         * @param base_env_resist Will override the base environmental
+         * @param resist_value Will override the base environmental
```

Comment:
Because that parameter can now use values other than the item's manually-set environmental resistance, depending on context. The other versions of the resist() function in the same file uses resist_value to indicate the same thing, so I changed the name for clarity and parity.